includes/classes/class-wcal-dashboard-report.php

```diff
@@ -452,7 +452,8 @@ public static function wcal_get_recovered_amount() {
 				self::$recovered_count = count( $ids );
 
 				foreach ( $ids as $order_id ) {
-					$order_total = (float) get_post_meta( $order_id, '_order_total', true );
+					$order       = wc_get_order( $order_id );
+					$order_total = (float) $order->get_total();
```

Comment:
This should be written as `$order_total = $order ? (float) $order->get_total() : 0;`

Similar checks should be done at both the places where code has been updated in this file.